apps/src/aichat/views/TeacherOnboardingModal.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import Button, {buttonColors} from '@cdo/apps/componentLibrary/button/Button';
+import Link from '@cdo/apps/componentLibrary/link/Link';
+import {
+  BodyOneText,
+  BodyTwoText,
+  EmText,
+  Heading3,
+  StrongText,
+} from '@cdo/apps/componentLibrary/typography';
+import AccessibleDialog from '@cdo/apps/sharedComponents/AccessibleDialog';
+import i18n from '@cdo/locale';
+
+import aichatI18n from '../locale';
+
+import moduleStyles from './onboarding-modal.module.scss';
+
+/**
+ * Renders a modal that warns the user to chat responsibly with AI.
+ */
+
+export interface TeacherOnboardingModalProps {
+  onClose: () => void;
+}
+
+const TeacherOnboardingModal: React.FunctionComponent<
+  TeacherOnboardingModalProps
+> = ({onClose}) => {
+  return (
+    <AccessibleDialog
+      onClose={onClose}
+      className={moduleStyles.teacherOnboardingModal}
+    >
+      <div className={moduleStyles.headerContainer}>
+        <Heading3>{aichatI18n.welcomeToAichatLab()}</Heading3>
+      </div>
+      <hr />
+      <div className={moduleStyles.contentContainer}>
+        <div className={moduleStyles.warning}>
+          <BodyTwoText>
+            {aichatI18n.teacherOnboardingModalWarning()}
+          </BodyTwoText>
+        </div>
+        <div className={moduleStyles.textContainer}>
+          <BodyOneText>{aichatI18n.chatComponentsHeader()}</BodyOneText>
+          <ul>
+            <li>
+              <BodyTwoText>
+                <StrongText>{aichatI18n.instructions()}</StrongText>
+                {aichatI18n.colonSpace()}
```

Comment:
It might be good to check with platform/international on the right way to localize this. I'm not sure if it's better to just include the entire string with the colon, and have the translators decide how to handle it, or if it's okay to localize the colon separately?

Reply:
Oh yeah - good call. Will do.

Reply:
I remembered why I did this. Design was that the component game is boldtyped, '(optional)' is italicized, and description is neither.

Reply:
I don't think we should split it quite that much.  `colonSpace` in particular seems challenging to localize in isolation.  This looks like a great candidate for a single piece of markdown.

Reply:
Thanks - confirmed already with @mgc1194 in https://codedotorg.slack.com/archives/CFTFD6BPV/p1728077404465749

Reply:
Used `<SafeMarkdown>` but losing the styling of Typography's `<BodyTwoText>`. Other than adding the styling from ` typography.module.scss` wondering if there's a way to maintain DSCO typography style?

![Screenshot 2024-10-04 at 5 42 33 PM](https://github.com/user-attachments/assets/b99cd6c7-bc39-41d7-97d3-6725e35d4163)

Reply:
I am going to localize the bullets in the list in a follow-up PR and include the styling needed. https://codedotorg.atlassian.net/browse/LABS-650